### DISCUSSION TITLE
pls help me: bash: dnsx: command not found!

### DISCUSSION THREAD

**Discussion by phanthuthao** [projectdiscovery/dnsx#22] [Q&A] [answered]
Monkey:/usr/local/bin$ sudo subfinder -silent -d yahoo.com | dnsx
 bash: dnsx: command not found!

**Comment by ehsandeep** [comment, marked as answer]
Hey @phanthuthao, as the error says, either you don't have `dnsx` installed on your system that you can do with `GO111MODULE=on go get -v github.com/projectdiscovery/dnsx/cmd/dnsx` or you have something wrong with go setup.
